src/main/java/com/google/devtools/build/remote/client/LogParserUtils.java

```diff
@@ -236,6 +261,29 @@ private void printEntriesInOrder(OutputStream outStream) throws IOException, Par
     }
   }
 
+  /**
+   * Prints each entry out individually (ungrouped) and a message at the end for how many entries
+   * were printed/skipped.
+   */
+  private void printEntriesInJson(OutputStream outStream) throws IOException, ParamException {
+    try (InputStream in = openGrpcFileInputStream()) {
+      PrintWriter out =
+          new PrintWriter(new BufferedWriter(new OutputStreamWriter(outStream, UTF_8)), true);
```

Comment:
You aren't using this `out`, not that I'd recommend that you do so.

Reply:
Thanks, I'll leave & use `out` as it will be consistent with the other methods